linux-errno/src/lib.rs

```diff
@@ -4,8 +4,16 @@
 #![no_std]
 #![deny(clippy::all)]
 
+// ErrNo values generated with:
+//
+// ```
+// bindgen /usr/include/errno.h \
+//   | sed -rn 's|pub const (E[A-Z0-9_]*): u32 = ([0-9]+);|\1 = \2,|p' \
+//   | sort
```

Comment:
There is no objective basis for this, but would you consider sorting on the integer value instead of the symbol name:

```
| sort -g -t "=" -k 2
```